autopush/router/fcm.py

```diff
@@ -32,8 +31,87 @@ def __init__(self, ap_settings, router_conf):
         self.auth = router_conf.get("auth")
         self.metrics = ap_settings.metrics
         self._base_tags = []
+        self.creds = {}
+        self.regid = ""
+        self.reasonTable = {
+            "MissingRegistration": {
+                "msg": ("'to' or 'registration_id' is blank or"
+                        " invalid: {regid}"),
+                "err": 500,
+                "errno": 1,
+            },
+            "InvalidRegistration": {
+                "msg": "registration_id is invalid: {regid}",
+                "err": 410,
+                "errno": 105,
+                "crit": True,
```

Comment:
I'd argue these are more error log level vs critical as they're all somewhat expected. fcm.py already has 1 log.critical call for one case, I'll argue that that one switch to error as well.